tools/gpstest/LimeUtil.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+LD_PRELOAD=lib/libLimeSuite.so bin/LimeUtil $@
```

Comment:
why are these checked in? we generally don't check in binaries unless there's a good reason. do we just need to clone and build a separate project?

Reply:
the LimeSuite repo is bigger, so I thought makes sense to prebuilt it once, but changed